packages/api-reference/src/features/BaseUrl/ServerForm.vue

```diff
@@ -39,11 +39,15 @@ function updateVariable(name: string, value: string) {
 
 <template>
   <div v-if="servers?.length">
-    <span class="server-form-title">Base URL</span>
+    <span
+      aria-label="Base URL"
+      class="server-form-title"
+      >Base URL</span
+    >
     <div class="server-form">
       <div class="server-form-container">
         <!-- Dropdown -->
-        <div class="server-item">
+        <div>
```

Comment:
I think the selector for base url needs the aria label, or maybe aria-labelled-by. I was trying my screen reader on one of the netlify previews for this PR and I did not hear the label announced when I navigated over BaseURL and the Client Libraries menus.

Reply:
hey @kayla-beckham, great feedbacks you are right! i kept it as a draft as I haven't got the time to go through all the screen reader support yet, but it will be def done in that pr :)

Reply:
Updated, I'll take a quick swing at the other blocks too